backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/reflist/RefList.java

```diff
@@ -20,29 +20,31 @@
  * #L%
  */
 
-package de.metas.serviceprovider.external.issuedetails;
+package de.metas.reflist;
 
-import lombok.AllArgsConstructor;
-import lombok.Getter;
+import de.metas.organization.OrgId;
+import lombok.Builder;
 import lombok.NonNull;
+import lombok.Value;
 
-import java.util.Optional;
-import java.util.stream.Stream;
+import javax.annotation.Nullable;
 
-@AllArgsConstructor
-@Getter
-public enum ExternalIssueDetailType
+@Value
+@Builder
+public class RefList
```

Comment:
see org.adempiere.ad.service.IADReferenceDAO.ADRefListItem.
might be what u want.

also pls note that usually, for names we use ITranslatableString instead of plain Strings.